.github/workflows/pull_request_ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI Check for Mobile-Wallet
```

Comment:
This is exactly the same except for when to trigger the action, right? Just add the pull_request alongside push in the last file. 
on:
  pull_request:
    branches:
      - 'master'
  push:
    branches:
      - 'master'